docs/export-map.rst

```diff
@@ -0,0 +1,163 @@
+export-map
+==========
+
+.. dfhack-tool::
+    :summary: Export fortress map tile data to a JSON file
+    :tags: dev map
+
+WARNING - This command will cause the game to freeze for minutes depending on
+map size and options enabled.
+
+Exports the fortress map tile data to a JSON file. (does not include items,
+characters, buildings, etc.) Depending on options enabled, there will be a
+``KEY`` table in the JSON with relevant [number ID] values that match a number
+to their object type.
+
+Usage
+-----
+
+::
+
+    export-map [include|exclude] [<options>]
+
+Examples
+--------
+
+``export-map``
+    Exports the fortress map to JSON with ALL data included
```

Comment:
Here and below: `Exports` -> `Export`. These are descriptions of what the command represents, not a description of what happens. Also, please end these sentences in a period.